benchmarks-mpp/commonMain/src/benchmarks/immutableList/Get.kt

```diff
@@ -18,7 +17,7 @@ open class Get {
 
     private var persistentList: PersistentList<String> = persistentListOf()
 
-    @Setup(Level.Trial)
+    @Setup
```

Comment:
Benchmark plugin does't supports different setup levels, all setup methods have `Trial` level there. 
JMH default setup level is also "Trial".